tests/nexus_test.py

```diff
@@ -0,0 +1,306 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Should we try (maybe as a pair-programming thing?) to test this on the loki workflow?

Reply:
Good idea!

Reply:
We tested and it works nicely: see https://github.com/scipp/esssans/pull/114